src/swagger-specs/streaming-ingestion.yml

```diff
@@ -0,0 +1,638 @@
+        required: true
+        schema:
+          type: string
+          enum:
+          - application/json
+          - application/x-jackson-smile
+      - name: CONNECTION_ID
+        in: path
+        description: The ID of the streaming connection you want to stream data to.
+        required: true
+        schema:
+          type: string
+      requestBody:
+        description: The data that you want to ingest to Platform.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/BatchMessage'
+        required: true
+      responses:
+        207:
+          description: A multi-status code. The recipient needs to consult the contents
+            of the multi-status response body for further information about the success
+            or failure of the method execution.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BatchResponseDTO'
+        400:
+          description: There was a problem with the request. See the response body
+            for a more specific error message.
+          content: {}
+        401:
+          description: Access is unauthorized. The user needs to provide a valid bearer
+            token.
+          content: {}
+        403:
+          description: Access is forbidden.
+          content: {}
+        413:
+          description: The payload is larger than 1 MB.
+          content: {}
+        429:
+          description: Too many requests. The service is experiencing high loads.
+          content: {}
+        500:
+          description: An internal server error has occurred.
+          content: {}
+        503:
+          description: The service is currently unavailable. Clients should retry
+            at least three times using an exponential back-off strategy.
+          headers:
+            Retry-After:
+              description: Specifies the number of seconds to retry after.
+              schema:
+                type: integer
+          content: {}
+      x-codegen-request-body-name: body
+components:
+  schemas:
+    streamingMessage:
+      required:
+        - header
+        - body
+      type: object
+      description: The payload to be published to Platform.
+      properties:
+        header:
+          $ref: '#/components/schemas/messageHeader'
+        body:
+          $ref: '#/components/schemas/messageBody'
+      example:
+        header:
+          schemaRef:
+            id: https://ns.adobe.com/{TENANT_ID}/schemas/{SCHEMA_ID}
+            contentType: application/vnd.adobe.xed-full+json;version={SCHEMA_VERSION}
+          imsOrgId: '5C1328435BF324E90A49402A@AdobeOrg'
+          datasetId: '5da9452f7de80400007jc52a'
+          createdAt: "1526283801869"
+          flowId: "69f188b6-af07-4587-8061-e94d34637b41"
+          source:
+            name: 'Google BigQuery Base Connection'
```

Comment:
Google BigQuery is a batch source, maybe just go with "ACME Streaming Connection" for the name here. (Could use some of the newer streaming sources too, but that could lead to some confusion with flow service)